Sources/CardPayments/Models/CardResult.swift

```diff
@@ -8,4 +8,8 @@ public struct CardResult {
 
     /// The order ID associated with the transaction
     public let orderID: String
+
+    //// :nodoc: This is the deep link url returned from 3DS authentication
+    // TODO: parse contents of this URL once we are clear on values returned from 3ds
+    public let deepLinkURL: URL?
```

Comment:
We'll want to update this repo to use the new `@_documentation(visibility: private)` syntax introduced in Xcode 14.3 (see BT PR - https://github.com/braintree/braintree_ios/pull/1034)